test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -2,7 +2,7 @@ const distributeSearch = require('../../../lib/simplifyExpression/distributeSear
 
 const TestUtil = require('../../TestUtil');
 
-function testDistributeMinus(exprStr, outputStr) {
+function testDistributeResult(exprStr, outputStr) {
```

Comment:
yes! 

We had a big change go through recently that changed a lot of the tests and not all of them were named well- I thought I caught and fixed most of them but I should probably do a pass. 

thanks :)